doc/060_forget.rst

```diff
@@ -195,6 +195,8 @@ The ``forget`` command accepts the following parameters:
     boundaries and not relative to when you run the ``forget`` command. Weeks
     are Monday 00:00 -> Sunday 23:59, days 00:00 to 23:59, hours :00 to :59, etc.
 
+.. note:: Specifying ``--keep-tag ""`` will match untagged snapshots only.
```

Comment:
Might be good to make this single quotes to be coherent with the examples below.